README.md

```diff
@@ -1,7 +1,28 @@
-# notation-core-go
+# Notation-core-go library
```

Comment:
Why are we making this change? I think we can keep the title to be same as the repo's name, i.e. `notation-core-go`.